src/solver/simulation/solver.hxx

```diff
@@ -1581,8 +1582,8 @@ void ISimulation<Impl>::loopThroughYears(uint firstYear,
         qs.wait(Yuni::qseIdle);
         qs.stop();
 
-        // At this point, the first set of parallel year(s) was run
-        if (!pFirstSetParallelWasRun)
+        // At this point, the first set of parallel year(s) was run with at least one year performed
+        if (!pFirstSetParallelWasRun && yearPerformed)
```

Comment:
- Ok, **pFirstSetParallelWasRun** has a bad name, and it should be replaced by its opposite : isFirstParallelSet, more clear (not sure it can be done within the current PR).
- We should even have a variable **isFirstParallelSetWithAPerformedYear**
- At least, if we should rename **pFirstSetParallelWasRun** as **pFirstSetParallelWithAPerformedYearWasRun**

Reply:
For now, not sure we need an extra variable **yearPerformed**, we can use what already exist